data/strings/strings.txt

```diff
@@ -2405,7 +2405,7 @@
     pl = Wypoczynek z dziećmi
     pt = Passeios com crianças
     pt-BR = Feriados em família
-    ro = Timp liber
+    ro = Divertisment
```

Comment:
... cu familia / cu copiii ? Keep in mind we also have [entertainment]. How about we also change "it" to "Divertimento in famiglia", it's surely better than "Tempo libero"

Reply:
> ... cu familia / cu copiii ? Keep in mind we also have [entertainment]. How about we also change "it" to "Divertimento in famiglia", it's surely better than "Tempo libero"

totally forgot about that

Reply:
Please don't forget that all these changes should be also reflected in data/categories.txt file (see comment = above):
`comment = Search category for water park/disneyland/playground/toys store; any changes should be duplicated in categories.txt @children!`